File: beginner/java/1180.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int n, i, menor = 0, posicao = 0;

        n = scanner.nextInt();

        int[] v = new int[n];

        for (i = 0; i < n; i++) {
            v[i] = scanner.nextInt();
            if (i == 0 || v[i] < menor) {
                menor = v[i];
                posicao = i;
            }
        }

        System.out.printf("Menor valor: %d\n", menor);
        System.out.printf("Posicao: %d\n", posicao);
        
        scanner.close();
    }
}

File: beginner/c/1180.c
#include <stdio.h>

int main() {
    int n, i, menor, posicao = 0;

    scanf("%d", &n);

    int v[n];

    for (i = 0; i < n; i++) {
        scanf("%d", &v[i]);
        if (i == 0 || v[i] < menor) {
            menor = v[i];
            posicao = i;
        }
    }

    printf("Menor valor: %d\n", menor);
    printf("Posicao: %d\n", posicao);

    return 0;
}
